src/utils/trigger-input-change-event.js

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+/* global window */
+
+import createEvent from './create-event.js';
```

Comment:
please co-locate these utility functions until they are used in more than one place in the codebase. the indirection is unnecessary when coupled to the input component